app/error_messages.py

```diff
@@ -40,3 +40,6 @@
     "Variants have more than one question type for block."
 )
 PREVIEW_WITHOUT_INTRODUCTION_BLOCK = "No introduction block found. Introduction block is mandatory when using the preview questions feature."
+LIST_REFERENCED_BEFORE_ADDED = (
+    "List '{list_name}' referenced as source before it has been created."
```

Comment:
Quick question - could the list ID be passed in as a key/val to the error message instead, to prevent using string formatting?

Reply:
Yeah, we can do it both ways, up to the devs which way we want this.

Reply:
I moved it to a key/val and lets see if other devs like it.